### PR TITLE
UI/become an instructor

### LINKED ISSUE #18: [FEAT]: ui/home - top Instructor
### Summary
NA
### Answer following questions?
1. Is your feature related to FrontEnd? Yes
2. Is your feature related to Backend? No
3. Is your feature related to API Integration? No
### Resources
image

### LINKED ISSUE #19: [FEAT]: ui/home - trusted companies
### Summary
NA
### Answer following questions?
1. Is your feature related to FrontEnd? Yes
2. Is your feature related to Backend? No
3. Is your feature related to API Integration? No
### Resources
image

### LINKED ISSUE #20: [FEAT]: ui/home - become a instructor
### Summary
NA
### Answer following questions?
1. Is your feature related to FrontEnd? Yes
2. Is your feature related to Backend? No
3. Is your feature related to API Integration? No
### Resources
image

### PR DESCRIPTION
**Pull request by Vinay-Pratap-Singh**
### Describe your changes

Write here...

### Issue ticket number and link

#18 #20 #19 

### Please confirm the below message,

- [x] I confirm that I have performed a self-review of my code, and written explainable comments in the code, the UI is rendered without any errors, the UI is responsive, dark mode UI is implemented, removed all debug-related code.

### CODE REVIEW

**Comment by Vinay-Pratap-Singh** [comment]
@sub1120 @shivamvijaywargi 
Can anybody review and merge it so that I can work on it?

**Comment by sub1120** [comment]
> @sub1120 @shivamvijaywargi Can anybody review and merge it so that I can work on it?

Acknowledged @Vinay-Pratap-Singh, I will review it soon. 

Sorry for the delay, but I see you have not assigned the reviewer in PR. It will be helpful if you assign anyone of us from next time, so we will get notification.

**Comment by Vinay-Pratap-Singh** [comment]
> > @sub1120 @shivamvijaywargi Can anybody review and merge it so that I can work on it?
> 
> Acknowledged @Vinay-Pratap-Singh, I will review it soon.
> 
> Sorry for the delay, but I see you have not assigned the reviewer in PR. It will be helpful if you assign anyone of us from next time, so we will get notification.

I cannot assign to any of you by myself, I checked but did not get any option for it.

**Comment by sub1120** [comment]
> > > @sub1120 @shivamvijaywargi Can anybody review and merge it so that I can work on it?
> > 
> > 
> > Acknowledged @Vinay-Pratap-Singh, I will review it soon.
> > Sorry for the delay, but I see you have not assigned the reviewer in PR. It will be helpful if you assign anyone of us from next time, so we will get notification.
> 
> I cannot assign to any of you by myself, I checked but did not get any option for it.

No problem, will check it.

File: src/app/page.tsx
Comment (sub1120) [inline src/app/page.tsx:1]: Vinay, could you please add last missing line in UI,

image

File: src/app/page.tsx [hunk 1]
```diff
@@ -132,6 +134,118 @@ export default function Home() {
...
+
+      {/* top instructor of the month */}
+      <section className="relative -top-40 mx-2 flex flex-col items-center justify-center space-y-5 border-2 border-gray-100 bg-white p-2 dark:bg-gray-800 md:mx-11 md:p-5 lg:mx-10 lg:p-10">
+        <h1 className="text-center text-lg font-semibold dark:text-white md:text-xl lg:text-2xl">
+          Top instructor of the month
+        </h1>
+
+        {/* container for top mentors */}
+        <div className="grid grid-cols-2 gap-2 sm:grid-cols-2 sm:gap-5 md:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5">
+          {Array.from({ length: 5 }).map((_, index) => {
```
Comment (sub1120) [inline src/app/page.tsx]: Vinay, I had added 5 users in `sampleData.ts` with there profile pic, please use those to dynamically map here.

File: src/app/page.tsx
Comment (sub1120) [inline src/app/page.tsx:1]: Vinay, this gap between section is big please reduce it, and give consistent gaps between different sections.

image

**Comment by sub1120** [comment]
@Vinay-Pratap-Singh, I am done with review. Nice work on top instructor section. Please check the comments.

File: src/app/page.tsx
Comment (Vinay-Pratap-Singh) [reply src/app/page.tsx:1]: done

File: src/app/page.tsx (hunk 1 above)
Comment (Vinay-Pratap-Singh) [reply src/app/page.tsx]: done

File: src/app/page.tsx
Comment (Vinay-Pratap-Singh) [reply src/app/page.tsx:1]: done

File: src/app/page.tsx (hunk 1 above)
Comment (Vinay-Pratap-Singh) [reply src/app/page.tsx]: done

File: src/app/page.tsx
Comment (Vinay-Pratap-Singh) [reply src/app/page.tsx:1]: done

**Comment by Vinay-Pratap-Singh** [comment]
> @Vinay-Pratap-Singh, I am done with review. Nice work on top instructor section. Please check the comments.

done from my end have a look @sub1120

**Review by sub1120 — APPROVED** [review]
All Good! Nice work Vinay.

**Pull request merged**